source/blog/2019-09-10-release-notes.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: post
+title: September 2019 Release Notes - Version 19.9.0
```

Comment:
let's add an emoji to the title of the release notes!  we used to do it pretty consistently, see https://finagle.github.io/blog/2019/03/14/release-notes/

Reply:
Thanks, updated!